706.js

```diff
@@ -0,0 +1,102 @@
+// Time Complexity : 
+// Space Complexity :
```

Comment:
Ideally your run time is expected to be constant and its O(1) amortized. It gets better with more complex hash functions. Space is O(n) - you somehow have to store all elements